### PR TITLE
SPLAT-1423: Added static ip support to CAPI installer.

### PR DESCRIPTION
**Pull request by vr4manta**
[SPLAT-1423](https://issues.redhat.com/browse/SPLAT-1423)

## Changes
- Added logic to inject static IP of masters and bootstrap when installing using CAPI instead of Terraform

## Prerequisites
- https://github.com/openshift/installer/pull/7501

## Blocking
- https://github.com/openshift/release/pull/49335

### CODE REVIEW

File: pkg/asset/machines/vsphere/capimachines.go [hunk 1]
```diff
@@ -79,24 +81,38 @@ func GenerateMachines(ctx context.Context, clusterID string, config *types.Insta
...
+				if err != nil {
+					return nil, fmt.Errorf("unable to get static ip config for machine %v: %w", machine.Name, err)
+				} else {
+					customVMXKeys["guestinfo.afterburn.initrd.network-kargs"] = kargs
+				}
+			}
+			capvNetworkDevices = append(capvNetworkDevices, deviceSpec)
 		}
 
+		fmt.Printf("Extra Properties %v\n", customVMXKeys)
```
Comment (rvanderp3) [inline pkg/asset/machines/vsphere/capimachines.go]:
```suggestion
		logrus.Debugf("Extra Properties %v\n", customVMXKeys)
```

**Review by rvanderp3 — COMMENTED** [review]
other than a nit, LGTM

File: pkg/asset/machines/vsphere/capimachines.go (hunk 1 above)
Comment (vr4manta) [reply pkg/asset/machines/vsphere/capimachines.go]: sorry, meant to remove that.  fixed.

**Comment by vr4manta** [comment]
/assign @patrickdillon
Ready for review.  Thanks!

**Comment by patrickdillon** [comment]
> /label capi

I looked into adding the label in prow config but labels like
/label platform/vsphere

seem to be defined as `additional_labels` and don’t seem configurable at the repo level. Need to follow up with dptp 

anyway this looks good!
/approve

**Review by patrickdillon — APPROVED** [review]

**Pull request merged**